README.md

```diff
@@ -22,43 +22,43 @@ these and other parameters are configurable through a json file Read more about
 For example, this `eth-network-params.json` adds a second node, running a different EL/CL client configuration.
 ```json
 {
-	"participants":[{
-        "el_client_type":         "geth",
-        "el_client_image":        "",
-        "el_client_log_level":    "",
-        "cl_client_type":         "lighthouse",
-        "cl_client_image":        "",
-        "cl_client_log_level":    "",
-        "beacon_extra_params":    [],
-        "el_extra_params":        [],
-        "validator_extra_params": [],
-        "builder_network_params": null
-	},{
-        "el_client_type":         "erigon",
-        "el_client_image":        "",
-        "el_client_log_level":    "",
-        "cl_client_type":         "nimbus",
-        "cl_client_image":        "",
-        "cl_client_log_level":    "",
-        "beacon_extra_params":    [],
-        "el_extra_params":        [],
-        "validator_extra_params": [],
-        "builder_network_params": null
-	}],
-	"network_params":{
-		"preregistered_validator_keys_mnemonic": "giant issue aisle success illegal bike spike question tent bar rely arctic volcano long crawl hungry vocal artwork sniff fantasy very lucky have athlete",
-		"num_validator_keys_per_node": 64,
-		"network_id": "3151908",
-		"deposit_contract_address": "0x4242424242424242424242424242424242424242",
-		"seconds_per_slot": 12,
-		"slots_per_epoch": 32,
-		"genesis_delay": 120,
-		"capella_fork_epoch": 5
-	},
-	"wait_for_finalization": false,
-	"wait_for_verifications": false,
-	"verifications_epoch_limit": 5,
-	"global_client_log_level": "info"
+  "participants":[{
+    "el_client_type":         "geth",
+    "el_client_image":        "",
+    "el_client_log_level":    "",
+    "cl_client_type":         "lighthouse",
+    "cl_client_image":        "",
+    "cl_client_log_level":    "",
+    "beacon_extra_params":    [],
+    "el_extra_params":        [],
+    "validator_extra_params": [],
+    "builder_network_params": null
+  },{
```

Comment:
I would suggest adding an in-line comment here and maybe even in `default-network-params.json` that each block of config corresponds 1:1 with a node. Otherwise, looks good to me.